vpr/src/base/vpr_types.h

```diff
@@ -1330,6 +1330,8 @@ struct t_placer_opts {
  *             an essentially breadth-first search, astar_fac = 1 is near   *
  *             the usual astar algorithm and astar_fac > 1 are more         *
  *             aggressive.                                                  *
+ * astar_offset: factor used to offset the expected future costs to         *
```

Comment:
"offset that is subtracted from the lookahead (expected future costs) in the timing-driven router."